### ISSUE TITLE
Email Disclaimer Update needed - s.15(2)(c) update

### ISSUE THREAD

**Issue by benrfairless** [openaustralia/righttoknow#314]
Update the email disclaimer to expressly state that the Right to Know email address is the correct email address for the purposes of s.15(2)(c) of the FOI Act.

**Comment by mlandauer** [comment]
This is the current footer text in the outgoing emails: https://github.com/openaustralia/righttoknow/blob/production/lib/views/outgoing_mailer/_followup_footer.text.erb

@brfairless can you suggest some specific wording

**Comment by benrfairless** [comment]
Something to the effect of:

"This request is made by someone using the Right to Know website. Right to Know is not making this request, simply assisting the public to make their own requests. 

The unique email address satisfies s.15(2)(c) of the Freedom of Information Act as an address valid to send notices to the person making this request."

Obviously, it would need to be "branded" to the language of Right to Know

**Comment by henare** [comment]
@brfairless - I really like the idea of pointing out to authorities that we're not making the request but an individual is. So often I see on agency response letterheads that they have put Right To Know as the organisation behind the request #duh

More broadly, I'm hesitant to add too many disclaimers and things to our footer. It's the kind of thing we'd cringe at from an agency and we'd be up to 5 paragraphs if we added these.

Can we think of a better way of achieving the same result we're after?

**Comment by bbqsrc** [comment]
The agencies themselves send 5 paragraph footers half the time so I don't see how it matters so much.

**Comment by bbqsrc** [comment]
Also, the _full_ footer only has to be sent with the initial email, not all future emails, so perhaps have a different footer for each case.

**Comment by henare** [comment]
> The agencies themselves send 5 paragraph footers half the time so I don't see how it matters so much.

Just because they do crap things doesn't mean we want to.

> Also, the full footer only has to be sent with the initial email, not all future emails, so perhaps have a different footer for each case.

Hey this is a good idea!

**Comment by equivalentideas** [comment]
This has been a real issue with the @detentionlogs adopt an incident function. I'd really like to see this addressed in the system, I think it is used as a way to create some friction in the process, confuse newcomers and deter a few requests.

Here's an example of the outcome for a Detailed Incident Report request: https://www.righttoknow.org.au/request/foi_request_for_detail_incident_151#incoming-1695

It would be interesting to try the footer text idea, maybe as Brendan is suggesting with only including it in the first message.

Another suggestion to through out there is that RTK could have an automated reply for that message (though I'd say this might be hard to catch).

I think we try including text to preempt this response in the first email and see if that has the desired effect. If not, plan B (come up with plan B).

What does everyone think of @brfairless text:

> "This request is made by someone using the Right to Know website. Right to Know is not making this request, simply assisting the public to make their own requests.
> 
> The unique email address satisfies s.15(2)(c) of the Freedom of Information Act as an address valid to send notices to the person making this request."

**Comment by bbqsrc** [comment]
@equivalentideas: it does work already. Asher's recent request preempts their statement and skips the bullshit.

Look at it less as a cumbersome pointless footer and more as necessary boilerplate to be considered a valid request under the legislation in all circumstances.

The "pre-footer" should basically consist as follows:

> This request is being made by an individual using the Right to Know website. The unique email address provided by the service for this request satisfies s.15(2)(c) of the Freedom of Information Act.

**Comment by equivalentideas** [comment]
Yeah that's great I recon. Totally agree about the footer. I think this is definitely worth adding in a sentence, because it potentially removes one back and forth coro with FOI Officer, making things a little easier for applicant.

Is it time for a pull request yet?

**Comment by benrfairless** [comment]
We could include it in the default sign off. Currently, when I create an FOI request to DIBP, the below is added at the end automatically:

```
Yours faithfully,

Ben Fairless
```

Perhaps we could expand on this to include

```
I am making this request using the Right to Know website. The unique email address provided by the service for this request satisfies s.15(2)(c) of the Freedom of Information Act.

Yours faithfully,

Ben Fairless
```

It would then be up to the user if they wanted to include it or not, we wouldn't have to increase the size of the disclaimer.

(On another topic) 5 paragraph signature blocks is nothing. I've seen a disclaimer from a private company that was 9 paragraphs and over 600 words!

**Comment by equivalentideas** [comment]
I don't think this should be left up to users. As long as it clarifies that it is a valid address they could still move the application to an alternative address if they wanted, but the default should make things simpler for all users.

**Comment by henare** [comment]
@brfairless The less things the user has to do the better so I don't like this option.

@equivalentideas I like @bbqsrc's wording too - PR time :thumbsup:

**Comment by equivalentideas** [comment]
I think it's Asher's wording :) should we ask her if she's happy for it to be in there?

**Comment by asherwolfie** [comment]
Yes, of course that would be fine :)

**Comment by henare** [comment]
Closed via #315

**Comment by equivalentideas** [comment]
We should keep an eye on how this works out and regroup if need be.

Onwards and upwards!
